src/modules/Experience/scenes/LoaderScene/LoaderScene.tsx

```diff
@@ -1,19 +1,38 @@
-import { Html, useProgress } from '@react-three/drei';
-import React from 'react';
+import { Html, shaderMaterial, useProgress } from '@react-three/drei';
+import React, { useEffect, useRef } from 'react';
 import { ChakraHtml } from '../../components';
 import { Text, useTheme } from '@chakra-ui/react';
+  // @ts-ignore
+  import loaderVertexShader from '../../shaders/LoaderShader/vertex.glsl';
+  // @ts-ignore
+  import loaderFragmentShader from '../../shaders/LoaderShader/fragment.glsl';
```

Comment:
```suggestion
  import {loaderFragmentShader,  loaderVertexShader} from '../../shaders/LoaderShader';
```
- You can have the 2 files in 1 import by generating an index.ts file. Check how I do it in other folders
- Lets avoid ts-ignore here. We can add the ignore in the tsconfig.